src/math/vec2.js

```diff
@@ -131,7 +131,7 @@ Object.assign(pc, (function () {
          * console.log("The two vectors are " + (a.equals(b) ? "equal" : "different"));
          */
         equals: function (rhs) {
-            return this.x === rhs.x && this.y === rhs.y;
+            return rhs && this.x === rhs.x && this.y === rhs.y;

```

Comment:
I would strongly discourage that change: what's the use case where the vector can be null?

Reply:
Compared Vector2 to null in c#, got an exception and forgot it's a struct in Unity... not needed now, removed.